### DISCUSSION TITLE
pyenv global setting is preserved on uninstallation

### DISCUSSION THREAD

**Discussion by anmol-aidora** [pyenv/pyenv#3048] [Q&A]
Even after I uninstalled pyenv and removed PYENV_ROOT directory, still, somehow, pyenv global setting is preserved and which python points to the newly installed and newly created PYENV_ROOT python directory. Is this expected behaviour?

**Comment by native-api** [comment]
It's unclear from the description what exactly you're doing and getting.
Could you provide diagnostic info as per https://github.com/pyenv/pyenv-virtualenv/blob/master/.github/ISSUE_TEMPLATE.md ?
